re/README.md

```diff
@@ -0,0 +1,51 @@
+# hello-reason
```

Comment:
Doesn't look like this will be needed, but we could port over some of the build steps to the root `README`